tools/rcverify.sh

```diff
@@ -216,3 +218,9 @@ echo $(tput setaf 6)
 echo run the following command to remove the scratch space:
 echo "  rm -rf '$DIR'"
 echo $(tput sgr0)
+
+if [ "$REMOVE_DIR" = true ]; then
```

Comment:
if removing directory maybe don't emit the message above? That is turn this into an if then else?

Reply:
Updated accordingly.